src/Microsoft.AspNet.Mvc.Razor/RazorPage.cs

```diff
@@ -46,16 +46,11 @@ public RazorPage()
             _writerScopes = new Stack<TextWriter>();
         }
 
-        public HttpContext Context
+        public HttpContext HttpContext
```

Comment:
If you feel like making the change (since we don't actually have a guideline or preference):

`public HttpContext HttpContext => ViewContext?.HttpContext;`